space-plugins/nuxt-base/server/api/_oauth.post.ts

```diff
@@ -0,0 +1,37 @@
+export default defineEventHandler(async (event): Promise<OAuthResponse> => {
+	const appConfig = useAppConfig();
+	const { initOAuth } = await readBody(event);
+	if (initOAuth) {
+		setCookie(event, AUTH_COOKIE_NAME, '', {
+			httpOnly: true,
+			secure: true,
+			sameSite: 'none',
+		});
+		return {
+			ok: false,
+			redirectTo: appConfig.auth.initOauthFlowUrl,
+		};
+	}
+
+	const appSession = await getAppSession(event);
```

Comment:
Either here or inside `getAppSession` there should be new code to retrieve the session based on the result from session token.